### PR TITLE
added noopener, fixed att-alert and html to markdown

### PR DESCRIPTION
**Pull request by itay1313**
DX-203

### CODE REVIEW

**Review by j9t — CHANGES_REQUESTED** [review]
Thanks Itay!

1) For the .attn code change: Let’s do this—but it begs the question what the status is for multi-paragraph notes. That is, the `{: .attn-foo}` only works with one paragraph—but we have seen that sometimes, people need more paragraphs, which only works with the `<div class=attn-foo>` code. Did you fix that, too (DX-202)?

2) Regarding `noopener noreferrer`, why would we add this to links to our own sites—we can be reasonably sure a user wouldn’t be exposed to an attack? But more importantly, it seems this is [less of an issue now](https://chromestatus.com/feature/6140064063029248), so let’s not add this.

**Comment by itay1313** [comment]
> Thanks Itay!
> 
> 1. For the .attn code change: Let’s do this—but it begs the question what the status is for multi-paragraph notes. That is, `{: .attn-foo}` only works with one paragraph—but we have seen that sometimes, people need more paragraphs, which only works with the `<div class=attn-foo>` code. Did you fix that, too (DX-202)?
> 2. Regarding `noopener noreferrer`, why would we add this to links to our own sites—we can be reasonably sure a user wouldn’t be exposed to an attack? But more importantly, it seems this is [less of an issue now](https://chromestatus.com/feature/6140064063029248), suggesting not to add this extra code.

The long, multi-paragraph notes don't look good anyway. I suggest adding several different notes in that case, like I did.

reverted the rel

File: pages/Documents/ClientSideConfiguration/ServerChatAPI/Methods/retrieve-pre-chat-survey.md [hunk 1]
```diff
@@ -45,7 +45,14 @@ Returns an XML or JSON response with the pre-chat survey.
 | visitorId | The current visitor ID. | alphanumeric | This parameter can be used to associate the survey to the current visitor.|
 | skill | Set a skill to the current visitor. | alphanumeric | |
 
-<div class="attn-note">To retrieve the <code>surveyApiId</code>, you need to call the Engagement and Window API. The information to retrieve the <code>domain</code> name can be found <a href="/agent-domain-domain-api.html">here</a>. Make sure to set the service name as <code>acCdnDomain</code>. <br><br>Through the Engagement API, you need to call <code>GET https://{domain}/api/account/{accountId}/configuration/le-campaigns/campaigns/{campaignId}/engagements/{engagementId}/revision/{revisionId}?v=3.0</code> to get the <code>windowId</code>. <br><br>Next, through the Window API, you need to call <code>GET https://{domain}/api/account/{accountId}/configuration/engagement-window/window-confs/{windowId}</code> to get the <code>surveyPreChatId</code>. The <code>surveyPreChatId</code> is the value that needs to be passed through the <code>surveyApiId</code> parameter.</div>
```
Comment (j9t) [inline pages/Documents/ClientSideConfiguration/ServerChatAPI/Methods/retrieve-pre-chat-survey.md:48]: This is one note, not three, and a use case for allowing HTML for multi-paragraph notes.

If you don’t want to ensure HTML support for notes right now, make this one large paragraph—but please take note to split this up again into several paragraphs once multi-paragraph support is ensured.

(Let me know if you have any questions or concerns.)

File: pages/Documents/Solutions/MobileAppMessaging/add-messaging-to-your-app.md [hunk 2]
```diff
@@ -24,15 +24,21 @@ When you add messaging to your app, you enable your consumers to communicate wit
...
+<img src="img/inapp2.png" alt="Add Messaging to Your App" style="max-width:230px;max-height:700px;">
+<img src="img/inapp3.png" alt="InApp" style="max-width:230px;max-height:700px;">
 
 ### Prerequisites
 
 To begin, you need:
 
 * Conversational Cloud account with messaging enabled.
 
-  <div class="attn-alert">Contact your LivePerson representative to enable messaging on your account.  <b>Tip:</b> If you see 'All Conversations’ when you log in, your account has messaging enabled.</div>
```
Comment (j9t) [inline pages/Documents/Solutions/MobileAppMessaging/add-messaging-to-your-app.md:35]: (Good catch here! This does look like an alert and a tip. Interesting combination :)

File: pages/Documents/WebsiteMonitoring/VisitInformationAPI/visit-information.md [hunk 3]
```diff
@@ -37,10 +37,11 @@ This API supports OAuth 1.0 authentication patterns and requires SSL protocol. P
 | v | API version  | string | Required |
 | sid | Session ID | string | Required |
 
-<div class="attn-note">
```
Comment (j9t) [inline pages/Documents/WebsiteMonitoring/VisitInformationAPI/visit-information.md:40]: As earlier, this is not several notes, but one. Make one paragraph if need be for the moment, but change again once multi-paragraph HTML markup for notes is supported.

**Review by j9t — CHANGES_REQUESTED** [review]
Nice and thorough work!

Two things that need attention.

File: pages/Documents/ClientSideConfiguration/ServerChatAPI/Methods/retrieve-pre-chat-survey.md (hunk 1 above)
Comment (itay1313) [reply pages/Documents/ClientSideConfiguration/ServerChatAPI/Methods/retrieve-pre-chat-survey.md:48]: fixed

File: pages/Documents/ClientSideConfiguration/ServerChatAPI/Methods/retrieve-pre-chat-survey.md [hunk 4]
```diff
@@ -45,7 +46,10 @@ Returns an XML or JSON response with the pre-chat survey.
...
+{: .attn-note}
```
[the remark this answers is not in the document]
Comment (itay1313) [reply pages/Documents/ClientSideConfiguration/ServerChatAPI/Methods/retrieve-pre-chat-survey.md:49]: I tested it, added a screenshot
image

File: pages/Documents/WebsiteMonitoring/VisitInformationAPI/visit-information.md [hunk 5]
```diff
@@ -37,10 +37,9 @@ This API supports OAuth 1.0 authentication patterns and requires SSL protocol. P
...
-<p>The <strong>visitorId</strong> and the <strong>sid</strong> parameters referenced above are retrieved from the Web Tag. Each visitor receives from the Tag their own <strong>visitorId</strong> (also called <strong>vid</strong>) and <strong>sid</strong> (also called <strong>Session ID</strong>) which you can then retrieve and pass to this API. For more information on these events, please see the Tag documentation <a href="/lp-tag-engagement-window.html">here</a>.</p>
-<p><b>This method of retrieving the parameters is relevant to monitored sessions only</b>. If you're trying to retrieve information on unmonitored sessions (for example, chat windows built by you using our APIs and not the default Conversational Cloud window), you'll need to use the <a href="/rt-interactions-app-engagement-overview.html">App Engagement API</a> instead, specifically the Create Session method which returns both of these parameters.</p>
-</div>
+{: .attn-note}
```
[the remark this answers is not in the document]
Comment (itay1313) [reply pages/Documents/WebsiteMonitoring/VisitInformationAPI/visit-information.md:40]: tested, all good here

File: pages/Documents/ConversationalAI/Third-PartyBots/IBMWatsonAssistant/basic-content.md [hunk 6]
```diff
@@ -475,10 +475,10 @@ Please note we only support **ONE ACTION** per response
 
 #### Transfer / Escalations
 
-<div class="attn-alert">
-<p><strong>Naming conventions:</strong> Before going into <strong>actions</strong> and <strong>skills</strong> is the naming convention between each. All non-escalation actions are defined by using underscores. For example, in the case of closing a conversation, the action name returned by <strong>Watson</strong> needs to be <strong>CLOSE_CONVERSATION</strong>.</p>
-<p>Further down the line, if any additional functionality is added that can be called by an action from the AI, it will follow the same naming convention. For escalations, the naming convention for these skills should use a "-" instead of whitespace. Furthermore, if transferring to a skill, specifically assigned to bots, it’s best practice to prefix the skill name with "BOT-" within Conversational Cloud.</p>
-</div>
+{: .attn-alert}
+**Naming conventions:** Before going into **actions** and **skills** is the naming convention between each. All non-escalation actions are defined by using underscores. For example, in the case of closing a conversation, the action name returned by **Watson** needs to be **CLOSE\_CONVERSATION**.
```
[the remark this answers is not in the document]
Comment (itay1313) [reply pages/Documents/ConversationalAI/Third-PartyBots/IBMWatsonAssistant/basic-content.md:479]: fixed

**Review by j9t — APPROVED** [review]

File: pages/Documents/ClientSideConfiguration/ServerChatAPI/Methods/retrieve-pre-chat-survey.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (j9t) [reply pages/Documents/ClientSideConfiguration/ServerChatAPI/Methods/retrieve-pre-chat-survey.md:49]: Thank you! Appreciated.

File: pages/Documents/WebsiteMonitoring/VisitInformationAPI/visit-information.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (j9t) [reply pages/Documents/WebsiteMonitoring/VisitInformationAPI/visit-information.md:40]: Thank you!

**Pull request merged**